x/ibc/07-tendermint/update.go

```diff
@@ -82,30 +85,17 @@ func checkValidity(
 			header.TrustedValidators, consState.NextValidatorsHash, tvalHash,
 		)
 	}
-	// assert trusting period has not yet passed
-	if currentTimestamp.Sub(consState.Timestamp) >= clientState.TrustingPeriod {
-		return sdkerrors.Wrapf(
-			types.ErrTrustingPeriodExpired,
-			"current timestamp minus the consensus state timestamp is greater than or equal to the trusting period (%s >= %s)",
-			currentTimestamp.Sub(consState.Timestamp), clientState.TrustingPeriod,
-		)
-	}
-
-	// assert header timestamp is not past the trusting period
-	if header.Time.Sub(consState.Timestamp) >= clientState.TrustingPeriod {
```

Comment:
Implicitly checked here: https://github.com/tendermint/tendermint/blob/v0.33.6/lite2/verifier.go#L183

Reply:
Please verify this is true @cwgoes

Reply:
yes, if the header timestamp is before now, and the trusting period has not expired relative to now, the trusting period has also not expired with respect to the header timestamp

we should ensure that our tests cover this case though

Reply:
Checked here: https://github.com/cosmos/cosmos-sdk/blob/aditya/misbehaviour-followup/x/ibc/07-tendermint/update_test.go#L146

Didn't remove any of the tests that corresponded to these checks when removing them since they are still done in lite.Verify